packages/remark-language-server/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "include": ["*.js"],
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "checkJs": true,
+    "declaration": true,
+    "lib": ["ES2020"],
+    "module": "ES2020",
+    "moduleResolution": "node",
+    "noEmit": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "target": "ES2020"
+  }
```

Comment:
Should use the main config, e.g.: https://github.com/remarkjs/remark/blob/main/packages/remark-parse/tsconfig.json